app/views/user_mailer/welcome_email.html.slim

```diff
@@ -0,0 +1,15 @@
+div style='background-color: #cfdfe4; border-radius: 20px; width: 75%; margin: 0 auto;'
+  h1 style='border-radius: 20px; text-align: center; font-weight: bold; height: 60px; padding-top: 30px;'
+    | Welcome to OSBB Application, #{@user.full_name}!
+  hr style='width: 90%; text-align: center;'
+  p style='margin: 20px;'
+    p style='width: 90%; margin: 5px 5%; font-size: 20px;'
+      | You have successfully signed up to frankivsk.osbb@gmail.com, 
```

Comment:
you dont sign up to an email but a web page
change to a link to website

Reply:
Add I18n for emails

Reply:
fixed